dora/core/server/master/src/main/java/alluxio/master/scheduler/MembershipManagerWorkerProvider.java

```diff
@@ -57,7 +59,7 @@ public List<WorkerInfo> getWorkerInfos() {
       if (mEnableDynamicHashRing) {
         return getLiveWorkerInfos();
       }
-      return mMembershipManager.getAllMembers();
+      return ImmutableList.copyOf(mMembershipManager.getAllMembers());
```

Comment:
`mMembershipManager.getAllMembers()` is immutable so you don't need a copy?

Reply:
`mMembershipManager.getAllMembers()` returns `WorkerClusterView` but here the return type has to be `List<WorkerInfo>`.